src/cellTagsTreeDataProvider.ts

```diff
@@ -23,13 +23,15 @@ export class TagTreeDataProvider implements vscode.TreeDataProvider<string> {
 		}
 	}
 
-	private registerEditorListeners(editor: vscode.NotebookEditor | undefined) {
+	private async registerEditorListeners(editor: vscode.NotebookEditor | undefined) {
 		this._editorDisposables.forEach(d => d.dispose());
 
 		if (!editor) {
 			return;
 		}
 
+		vscode.commands.executeCommand('setContext', 'jupyter:showTagsExplorer', true);
```

Comment:
saw you marked the function as `async`. Did you mean to also add `await` here?

Reply:
Thanks, will resolve this in next PR.